lib/linguist/languages.yml

```diff
@@ -1637,6 +1637,7 @@ KRL:
 KiCad:
   type: programming
   extensions:
+  - .kicad_pcb
   - .sch
```

Comment:
The first extension should be the primary extension. I think `.sch` should stay the primary one ;)
(It's not very well documented, sorry about that.)